CHANGES

```diff
@@ -1,3 +1,7 @@
+Deprecation Warning (2021-06-07)
+--------------------------------
+There will be a future change to where the frontend bundle will be loaded asynchronously. This will be a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.
+
```

Comment:
```suggestion
Unreleased
----------

### Deprecation Warning

There will be a future change to where the frontend bundle will be loaded asynchronously. This will be a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.

```